pontoon/machinery/utils.py

```diff
@@ -71,7 +77,7 @@ def get_google_generic_translation(text, locale_code):
         }
 
 
-def get_google_automl_translation(text, locale):
+def get_google_automl_translation(text, locale, format="text"):
```

Comment:
Because this gets called directly from [here](https://github.com/mozilla/pontoon/blob/05be2896ef22bceb052262e29eb67a2c1d6f0066/pontoon/machinery/automl_warmup.py#L41).

Reply:
Thanks for clarifying! I feel like having a default value in `get_google_generic_translation` would be more consistent, but that's non-blocking.